website/src/components/quickstartTOC/index.js

```diff
@@ -108,8 +109,24 @@ function QuickstartTOC() {
         }
       });
 
-      const quickstartTitle = document.querySelector("header h1");
-      quickstartTitle.classList.add(style.quickstartTitle);
+      // Get title by frontmatter title
+      let quickstartTitle = document?.querySelector("header h1");
+
+      // Get lifecycle badge from markdown content h1
+      // Example: # Demo h1 title <Lifecycle status="beta"/>
+      const markdownTitleLifeCycleBadge = document?.querySelector(
+        ".quickstart-container .step-container > h1 > span.lifecycle-badge"
+      );
+
+      if (quickstartTitle) {
```

Comment:
This is where the original issue was occurring where the page would crash locally when adding a markdown h1 to a guide page. `quickstartTitle` was undefined, so when `quickstartTitle.classList.add` was called, the page would crash.